ansible/roles/stack-sunbird/templates/sunbird_content-service.env

```diff
@@ -45,4 +45,7 @@ channel.default=in.ekstep
 
 content.cache.ttl=86400
 content.cache.read=true
-content.cache.hierarchy=true
\ No newline at end of file
+content.cache.hierarchy=true
+
+
+schema.base_path=https://sunbirddev.blob.core.windows.net/sunbird-content-dev/schemas/{{ env }}/
```

Comment:
blobname is hardcoded

Reply:
lets change the url later to a valid one @maheshkumargangula, merjing for testing purpose.